google-cloud-examples/pom.xml

```diff
@@ -68,6 +68,7 @@
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-securitycenter</artifactId>
+      <version>0.117.0</version>
```

Comment:
The examples should depend on the bom (or a BOM) if we want users to use the BOM for their own code.

Reply:
The examples project does import the google-cloud-bom. I'll remove this version and make sure it works